### DISCUSSION TITLE
I want to use CSSModules with Mantine, as it's the default in v7, but will I have issues if I use it in v6 too?

### DISCUSSION THREAD

**Discussion by andyjessop** [mantinedev/mantine#4495] [Q&A] [answered]
I'm starting a new project with Mantine v6 (because v7 isn't quite ready for primetime), but I want to use CSSModules for my custom styles. Will there be any conflicts with Mantine's internal styling, or any other issues?

**Comment by rtivital** [comment, marked as answer]
You can use any styling solution with Mantine v6 or any other version, there should not be any issues.

**Reply by andyjessop** [reply]
Great, thanks very much!
